UnitedIncome/components
# [APP-5216] Upload Trades - Scaffold

### Description
Changes to fix the interface for MessageCard, required for Upload Trade CSV

Part of four PR's:

API - https://github.com/UnitedIncome/api/pull/3157
Client - https://github.com/UnitedIncome/client/pull/4020
UI-common - https://github.com/UnitedIncome/ui-common/pull/490
Components - **This**

### Testing Instructions
See Client PR above for testing instructions.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-5126)
